woocommerce/woocommerce-admin
# Onboarding: Add offline payment methods

Fixes #3710 

Adds the offline payment methods (bank transfer and cash on delivery) to the payments task.

### Screenshots
<img width="700" alt="Screen Shot 2020-03-05 at 3 23 25 PM" src="https://user-images.githubusercontent.com/10561050/75990647-82a3e900-5ef5-11ea-81aa-577ac078dc4a.png">
<img width="727" alt="Screen Shot 2020-03-05 at 3 23 17 PM" src="https://user-images.githubusercontent.com/10561050/75990650-846dac80-5ef5-11ea-88fe-093ee9c47a15.png">


### Detailed test instructions:

1. Enable the task list and visit the payments task (delete the `woocommerce_task_list_payments` option if you need to reset).
1. Toggle the Cash on Delivery method and make sure that it affects the actual setting at `wp-admin/admin.php?page=wc-settings&tab=checkout`.
1. Delete bank transfer accounts if you already have any.
1. Set up the Direct Bank transfer option using any bank details (these are not validated in core either, but added a minimum of account number or IBAN here).
1. Save the configuration and make sure settings appear accurately at `wp-admin/admin.php?page=wc-settings&tab=checkout&section=bacs`.
